tools/mksymtab.sh

```diff
@@ -21,21 +21,45 @@
 
 export LC_ALL=C
 
-usage="Usage: $0 <imagedirpath> [symtabprefix [additionalsymbolspath]]"
+usage(){
+  if [ $# -ne 0 ]; then
+    echo "ERROR: $@" > /dev/stderr
+  fi
+  echo -e "\nUsage: ${BASH_SOURCE[0]} <imagedirpath> [symtabprefix] [-a additionalsymbolspath]" > /dev/stderr
```

Comment:
why add `> /dev/stderr` and use `${BASH_SOURCE[0]}` which isn't portable

Reply:
> why add `> /dev/stderr` and use `${BASH_SOURCE[0]}` which isn't portable

- `/dev/stderr` : I mistakenly thought that logs printed to /dev/stdout were being ignored in CI's error messages.
- `${BASH_SOURCE[0]}` : OK, the value of `$0` is still the script file name.


The above will be restored , thank you.